fedimint-cli/src/client.rs

```diff
@@ -581,7 +580,7 @@ pub async fn handle_command(
                 match update {
                     WithdrawState::Succeeded(txid) => {
                         return Ok(json!({
-                            "txid": txid.to_hex(),
+                            "txid": format!("{txid}"),
```

Comment:
so it's possible to incorrectly write `txid.encode_hex::<String>()`? If so we should try to do something to avoid such traps

Reply:
I implemented our `consensus_encode_to_hex` correctly and replaced the `format!` with it. There not much we can do about a foreign function incorrectly encoding a foreign type unfortunately.